azure/template.json

```diff
@@ -249,6 +252,34 @@
                     }
                 }
             }
+        },
+        {
+            "apiVersion": "2020-06-01",
+            "name": "[concat('role-assignments-', copyIndex())]",
```

Comment:
Can we change this to `[concat(variables('resourceNamePrefix'), 'role-assignments-', copyIndex())]` to make it easier to find in the logs

Reply:
Updated